pallets/orml-currencies-allowance-extension/src/default_weights.rs

```diff
@@ -1,55 +1,77 @@
 
-//! Autogenerated weights for `orml_currencies_allowance_extension`
+//! Autogenerated weights for orml_currencies_allowance_extension
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2023-07-19, STEPS: `100`, REPEAT: `10`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: 2023-07-24, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
 //! WORST CASE MAP SIZE: `1000000`
-//! HOSTNAME: `Hugos-MacBook-Pro.local`, CPU: `<UNKNOWN>`
-//! EXECUTION: None, WASM-EXECUTION: Compiled, CHAIN: Some("foucoco"), DB CACHE: 1024
+//! HOSTNAME: `Marcels-MBP`, CPU: `<UNKNOWN>`
```

Comment:
I'm curious about this. Since the benchmark result depends on the hardware where it runs, would it be a good idea to have a dedicated setup in a VM to run all the benchmarks?

Reply:
That's a very good point. I found [this](https://substrate.stackexchange.com/a/3940/2607) related comment in a forum post, and also [this](https://substrate.stackexchange.com/a/3944/2607) comment (in the same thread) that suggests using a dedicated VM for this purpose, as you suggest.

Running a separate node that meets [these](https://wiki.polkadot.network/docs/maintain-guides-how-to-validate-polkadot#standard-hardware) requirements just for generating the benchmarked weights might be overkill though. However, it's important that we keep this 'problem' in mind. 

@TorstenStueber do you have an opinion on this?

Reply:
A defined [another ticket](https://github.com/pendulum-chain/tasks/issues/107) to consider a better solution in the future.

Reply:
Perfect, thanks 👍